README.md

```diff
@@ -27,7 +27,28 @@ If you have a well phrased question that might benefit others, consider asking i
 
 ## Quick start:
 * Make sure you have golang version 1.6 or greater installed.
+* If you do not have it yet create your GOPATH and export it
+		
+		$ mkdir $HOME/work
```

Comment:
can you change the `work` to `golang`. That way it's more obvious that it's a dedicated dir.